src/prefect/infrastructure/provisioners/cloud_run.py

```diff
@@ -192,46 +221,45 @@ async def provision(self, work_pool_name: str, base_job_template: dict):
 
                             - Create GCP credentials block [blue]{work_pool_name}-push-pool-credentials[/] to store the service account key
                     """
-                ),
-                expand=False,
-            )
-            self._console.print(table)
+            ),
+            expand=False,
+        )
+        self._console.print(table)
+        if self._console.is_interactive:
             choice = Confirm.ask(
```

Comment:
nit: renaming `choice` to `confirm`? I think we've done that in `prefect deploy` anyway